elements/howto-checkbox/howto-checkbox.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * A `HowToCheckbox` represents a boolean option in a form. The most common type
+ * of checkbox is a dual-type which allows the user to toggle between two
+ * choices -- checked and unchecked.
+ */
+(function() {
+  /**
+   * Define key codes to help with handling keyboard events.
+   */
+  const KEYCODE = {
+    SPACE: 32,
+  };
+
+  /**
+   * The `HowToCheckbox` exposes a single `checked` attribute/property for
+   * toggling its state. Changes to the `checked` property will also be
+   * reflected to an `aria-checked` attribute. Similarly, the `disabled`
+   * property is reflected to an `aria-disabled` attribute. This controls
+   * whether the element is operable or not.
+   */
+  class HowToCheckbox extends HTMLElement {
+    static get observedAttributes() {
+      return ['checked', 'disabled'];
```

Comment:
So here’s a fun thing: Native checkboxes _don’t_ react to changes of their `checked` attribute, but they _do_ react to a change of `disabled`. Is that insanity that we want to make more consistent or do we want to mimic the native element as closely as possible?